src/main/java/org/example/Game2048/Point.java

```diff
@@ -0,0 +1,19 @@
+package org.example.Game2048;
+
+import lombok.Data;
+
+import javax.persistence.*;
+
+@Entity
+@Data
+public class Point {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Integer x;
+    private Integer y;
+    private Integer value;
+    @ManyToOne
+    private Board boardId;
```

Comment:
troche dziwna konwencja zeby nazywac to pole typu Board - boardid, przeciez to nie jest tylko id a caly obiekt :)